_includes/configuration/toolbar-mode.md

```diff
@@ -6,6 +6,25 @@
 
 The `toolbar_mode` option is used to extend the toolbar to accommodate the overflowing toolbar buttons. This option is useful for small screens or small editor frames.
 
+The editor will move toolbar buttons to the toolbar drawer based on:
```

Comment:
Should specify that this applies to floating and sliding modes and not the other two?

Reply:
May of been fixed by: https://github.com/tinymce/tinymce-docs/pull/1612/commits/85c06bb5d09d833d450f339a14642ad85c3ad1db